index.html

```diff
@@ -141,7 +141,7 @@ <h2>
         <p>
           The steps to <dfn>validate a URL-based payment method
           identifier</dfn> are given by the following algorithm. The algorithm
-          takes a <a data-cite="!URL#concept-url">URL</a> <var>url</var> as
+          takes a <a data-cite="!URL#concept-url">URL</a> as
           input and returns true if the URL is valid:
```

Comment:
This one is ok. The variable “url” is of type URL.

Reply:
@marcoscaceres I changed PR with your comment. Thanks!